rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+```
+
+Note that the above approach does not require all the data in a submitted DAPS to be of the same modality to be processed correctly. For example a DAPS at the project level can contain imaging and 10X datasets and these will be correctly processed in an independent manner. 
+
+Furthermore if part of the data submitted in a DAPS has already been processed the Analysis infrastructure will only re-process the parts that have not been modified.
+
+## Metadata Requirements for 10X V2 3’ scRNA-seq datasets
+The specific implementation of the above metadata requirements for 10X V2 3’ RNA-seq datasets is presented here as an initial use case and an example. 10X V2 3’ data are the most common type of data currently deposited in the DSS and therefore represent a widely applicable use case.
+
+Requirement (1) above can be satisfied by having a globally unique ‘Library ID’ variable in the metadata that identifies the Illumina sequencing library from which each input file originates from. More details on the requirements and a specific implementation of this solution can be found currently under consideration here: [PR](https://github.com/HumanCellAtlas/dcp-community/pull/87), [document](https://github.com/HumanCellAtlas/dcp-community/blob/mfreeberg-rfc-library-preparation/rfcs/text/0000-rfc-library-preparation.md).
+
+Requirement (2) can be satisfied in two alternative ways: (a) sufficient metadata can be provided as outlined in table 1 below to identify all file triplets, or (b) a globally unique triplet field can be used to identify file correspondence by the submitter. We propose the latter approach as collecting all the required metadata may not be feasible in all cases and places an undue burden on the submitter in the common scenarios where only one triplet is submitted.
+
+We propose that the metadata required to automatically identify the triplets (Table 1) is made optional and users are encouraged to submit these fields as it can help identify errors and also can in some cases provide important covariates for scientific analyses.
+
+**Table 1: Fields required for identification of fastq file correspondence**
+<table>
+  <tr>
+   <td><strong>Required Fields</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Justification</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Run Date and Time
+   </td>
+   <td>ISO Date and time at which the run was performed
+   </td>
+   <td>This is for the rare scenario where flowcells are re-used and ensures that each run is processed as if it were a separate flowcell
+   </td>
+  </tr>
+  <tr>
+   <td>Flowcell ID
+   </td>
+   <td>The Illumina identifier of the flowcell id
+   </td>
+   <td rowspan="3" >The unique combination of Flowcell ID, Lane and Barcode uniquely identifies any multiplexed library on an Illumina Flowcell
+   </td>
+  </tr>
+  <tr>
+   <td>Lane in Flowcell
+   </td>
+   <td>The lane in the flowcell
+   </td>
+  </tr>
+  <tr>
+   <td>Barcode of Library
```

Comment:
Why is the library barcode needed? we only take demuxed fastq files? how does that information help?